packages/firebase_messaging/README.md

```diff
@@ -110,6 +148,15 @@ By default background messaging is not enabled. To handle messages in the backgr
    <application android:name=".Application" ...>
    ```
 
+#### iOS configuration (Swift)
+1. In `AppDelegate.swift`, add the following:
```

Comment:
Note: `import firebase_messaging` needs to be added to the top of `AppDelegate.swift`. Worth mentioning.

Reply:
Good point! Easy to miss. I'll add that.